hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/CustomGasCharging.java

```diff
@@ -138,4 +174,13 @@ private void validateAndCharge(
         worldUpdater.collectFee(aPayer.hederaId(), aAmount);
         worldUpdater.collectFee(bPayer.hederaId(), bAmount);
     }
+
+    private void requireAllNonNull(
```

Comment:
Could retrofit this into `validateAndCharge` L169, `chargeWithRelayer` L138, `chargeWithOnlySender` L130, and `chargeForGas` L105

Reply:
After some thought, deleted this method and just did all the `requireNonNull()` "out-of-line" explicitly; should reduce risk of future PR friction. 😇